internal/evaluator/conftest_evaluator.go

```diff
@@ -149,6 +153,46 @@ type conftestEvaluator struct {
 	namespace     []string
 }
 
+type conftestRunner struct {
+	runner.TestRunner
+}
+
+func (r conftestRunner) Run(ctx context.Context, fileList []string) (result []output.CheckResult, data Data, err error) {
+	result, err = r.TestRunner.Run(ctx, fileList)
+	if err != nil {
+		return
+	}
+
+	// we can't reference the engine from the test runner or from the results so
+	// we need to recreate it, this needs to remain the same as in
+	// runner.TestRunner's Run function
+	var engine *conftest.Engine
+	engine, err = conftest.LoadWithData(r.Policy, r.Data, r.Capabilities, r.Strict)
+	if err != nil {
+		return
+	}
+
+	store := engine.Store()
+
+	var txn storage.Transaction
+	txn, err = store.NewTransaction(ctx)
+	if err != nil {
+		return
+	}
+
+	ids := []string{} // everything
+
+	var d any
+	d, err = store.Read(ctx, txn, ids)
+	if err != nil {
+		return
+	}
+
+	data, _ = d.(map[string]any) // if we fail here we don't have the data to return, and mostly we won't
```

Comment:
If an error happens, don't we want things to fail?

Reply:
This could only fail if we don't pass any data to Conftest or if the data passed
is not a `map[string]any`, with this in those cases `data` is `nil`. So this
prevents a panic in those cases.
Not passing any data to Conftest would mean that we didn't even pass effective
time.
We could err here, but I don't think that error could happen

Reply:
Personally, I'd rather propagate an error even if I don't think it's possible to happen. If I'm right, the error never shows up and it doesn't matter, but if I'm wrong, I'll have a chance of understanding what went wrong. Just my 2 cents. Not a blocker for this PR.

Reply:
I did end up propagating the error...